tests/unittest/ops/test_activation.py

```diff
@@ -352,6 +352,58 @@ def _test_celu(
         module.run_with_tensors([x1_pt], [x2])
         torch.testing.assert_close(x2, x2_pt, atol=1e-2, rtol=1e-2)
 
+    def _test_gelu(self, input_size, test_name="gelu", copy_op=False, dtype="float16"):
+        assert len(input_size) == 2
+        X1 = Tensor(
+            shape=[IntImm(input_size[0]), IntImm(input_size[1])],
+            dtype=dtype,
+            name="input0",
+            is_input=True,
+        )
+        X2_op = ops.elementwise(FuncEnum.GELU)
+        if copy_op:
+            X2_op = ops.elementwise(**X2_op._get_op_attributes())
+        X2 = X2_op(X1)
+        X2._attrs["is_output"] = True
+        X2._attrs["name"] = "output0"
+
+        target = detect_target()
+        module = compile_model(X2, target, "./tmp", f"{test_name}_{dtype}")
+        x1_pt = get_random_torch_tensor(input_size, dtype)
+        OP_pt = torch.nn.GELU()
+        x2_pt = OP_pt(x1_pt)
+
+        x2 = torch.empty_like(x2_pt)
+        module.run_with_tensors([x1_pt], [x2])
+        torch.testing.assert_close(x2, x2_pt, atol=1e-2, rtol=1e-2)
```

Comment:
the tolerance here seems a bit large?

Reply:
1e-2 seems to be the standard tolerance we use for testing activation ops. See the test file for examples